contrib/demo/clusters/kind/createKindClusters.sh

```diff
@@ -56,13 +56,20 @@ detect_podman() {
         return
     fi
 
+    if [[ "$OSTYPE" == "darwin"* && -z "$(podman ps)" ]]; then
+        # Podman machine is not started
```

Comment:
can you add some error message here?

Reply:
Good idea, added in 763f62bd90e8643cdce4e0d618a5f5ca584fc6d9.